ts/navigation/WalletNavigator.ts

```diff
@@ -74,6 +75,11 @@ const WalletNavigator = createStackNavigator(
     },
     [ROUTES.PAYMENT_HISTORY_DETAIL_INFO]: {
       screen: PaymentHistoryDetailsScreen
+    },
+
+    // Navigator section for Bonus section
+    [ROUTES.AVAILABLE_BONUS_LIST]: {
```

Comment:
```suggestion
    [ROUTES.BONUS_AVAILABLE_BONUSES]: {
```
Perhaps having BONUS prefix for all bonus screens could be better when we trace them in Mixpanel

Reply:
Yep, sounds good! I'm changing it.

Reply:
Maybe just `BONUS_AVAILABLE_LIST` could be less redundant?